diff --git a/node_modules/react-native/scripts/codegen/generate-artifacts-executor/generateReactCodegenPodspec.js b/node_modules/react-native/scripts/codegen/generate-artifacts-executor/generateReactCodegenPodspec.js
index 554dbca..129718b 100644
--- a/node_modules/react-native/scripts/codegen/generate-artifacts-executor/generateReactCodegenPodspec.js
+++ b/node_modules/react-native/scripts/codegen/generate-artifacts-executor/generateReactCodegenPodspec.js
@@ -15,8 +15,8 @@ const {
   TEMPLATES_FOLDER_PATH,
   packageJson,
 } = require('./constants');
-const {codegenLog} = require('./utils');
-const {execSync} = require('child_process');
+const { codegenLog } = require('./utils');
+const { execSync } = require('child_process');
 const fs = require('fs');
 const path = require('path');
 
